components/permalink_redirector/index.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {redirect} from './actions';
+import PermalinkRedirector from './permalink_redirector';
+
+const mapStateToProps = (state, ownProps) => {
```

Comment:
nit: `ownProps` vs. just `props` in this case? I usually see the former when we have to distinguish because we return a callback instead of a direct mapping.

Reply:
hmm not sure if i follow the comment 🤔 
distinguish between cases where we return callbacks in `mapStateToProps` vs direct mapping?

Reply:
Never mind, I was thinking there was a pattern as to when to call it `props` vs `ownProps`, but we don't have any pattern and I can't find any authority on this, so this is fine :)

Reply:
FWIW redux documentation calls it `ownProps`
https://react-redux.js.org/using-react-redux/connect-mapstate#defining-mapstatetoprops